hack/local-development/remote-garden/open-gardener-tunnels

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname "${0}")/../common/helpers
+
+QUIC_SERVER_IMAGE=ghcr.io/mvladev/quic-reverse-http-tunnel/quic-server:v0.1.2
+QUIC_CLIENT_IMAGE=ghcr.io/mvladev/quic-reverse-http-tunnel/quic-client-tcp:v0.1.2
```

Comment:
Should these images rather be moved into the gardener org/registry?

Reply:
Well the images are build with this https://github.com/mvladev/quic-reverse-http-tunnel/blob/master/.github/workflows/cd_workflow.yaml and stored in the github container registry. Those would be used for local development and are not fetched all the time.

I guess they can be added to `images.yaml` and be mirrored?

Reply:
The mirroring is only happening for productive images.

Reply:
As discussed, let's leave the images as they are for now.